src/monda/sorad/qc.py

```diff
@@ -43,7 +44,38 @@ def nearest(items, pivot):
      return nearest_value, nearest_index
 
 
-# Filters used in QC of l and e spectra (Step 1 in 3C and FP chain)
+# Filters used in Step (0) QC (based on tilt and relative azimuth)
+def rel_az_filter(rel_view_az, lower_azi_bound = 110, upper_azi_bound = 170):
+    """ Function to test that So-Rad relative viewing azimiuth is within given angular bounds. 
+    The default lower (110 deg) and upper (170 deg) bounds are based on the AMT cruise defaults"""    
```

Comment:
Remove reference to AMT which is not a standard procedure or expected to be general knowledge. 

Typically measurements at 90 degrees will be similar quality to 135, but going to 170 seems excessive given glint risk.

Reply:
I will redo as suggested